textarea/textarea_test.go

```diff
@@ -416,15 +419,210 @@ func TestVerticalNavigationShouldRememberPositionWhileTraversing(t *testing.T) {
 	}
 }
 
-func TestRendersEndOfLineBuffer(t *testing.T) {
-	textarea := newTextArea()
-	textarea.ShowLineNumbers = true
-	textarea.SetWidth(20)
-
-	view := textarea.View()
-	if !strings.Contains(view, "~") {
-		t.Log(view)
-		t.Fatal("Expected to see a tilde at the end of the line")
+func TestView(t *testing.T) {
+	tests := []struct {
+		name      string
+		modelFunc func(Model) Model
+		expected  string
+	}{
+		{
+			name: "placeholder",
+			expected: heredoc.Doc(`
+				>   1 Hello, World!
+				>   ~
+				>   ~
+				>   ~
+				>   ~
+				>   ~
```

Comment:
```suggestion
				>   1 Hello, World!
				>  ~
				>  ~
				>  ~
				>  ~
				>  ~
```

Reply:
The plan is to merge these tests then fix the current bug, then update the tests?